pyshelf/bucket_update/search_updater.py

```diff
@@ -78,7 +78,9 @@ def add_artifact_metadata(self, path, bulk_update):
         """
         identity = self.container.resource_identity_factory \
             .from_cloud_identifier(path)
-        updater = ArtifactMetadataUpdater(self.bucket_container, identity)
```

Comment:
It needed to be at one point then got changed. a502197b0f3fcd0cc05a85a65d266716c264d010